producer-consumer.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: details 
+title: Producer - Consumer 
+main_title: Producer - Consumer 
+pnet: pnet1
```

Comment:
define the net here,
```
net:
  transitions: ".."
  places: ".."
  arcs: ".."
```

then render into the `<script>` tag, using `{{ page.net.transition | jsonify }}` etc

Reply:
Done.